.github/PULL_REQUEST_TEMPLATE/release.md

```diff
@@ -2,4 +2,4 @@
 
 Before releasing a new version of elements: (check the boxes to acknowledge you've followed this template)
 
-- [ ] [Read the release section of `CONTRIBUTING.md`](../../CONTRIBUTING.md#releasing-elements)
+- [ ] [Read the release section of `CONTRIBUTING.md`](../../blob/main/CONTRIBUTING.md#releasing-elements)
```

Comment:
```suggestion
- [ ] [Read the release section of `CONTRIBUTING.md`](../../CONTRIBUTING.md#releasing-elements)
```

Reply:
Hm? I thought we wanted to fix the broken link 😅

Reply:
Does `../../blob/main/CONTRIBUTING.md#releasing-elements` go to a specific header in CONTRIBUTING.md for you? I was a 404 for me until I change it back to `../../`

Reply:
(The path needs to be relative from the PR, not from `.github/PULL_REQUEST_TEMPLATE/release.md`).

Reply:
Ah, yes! You are correct. It's been too long of a day. haha. Thanks again for updating this!